VENTokenField/VENBackspaceTextField.m

```diff
@@ -24,15 +24,12 @@
 
 @implementation VENBackspaceTextField
 
-- (BOOL)keyboardInputShouldDelete:(UITextField *)textField
-{
+- (void)deleteBackward {
```

Comment:
`keyboardInputShouldDelete` seemed to prevent `textField:shouldChangeCharactersInRange:replacementString` from being called on backspaces. `deleteBackward` doesn't seem to have the same limitation.